examples/msr-events.rs

```diff
@@ -0,0 +1,111 @@
+use clap::{App, Arg, ArgMatches};
+use colored::*;
+use env_logger;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::time::Instant;
+use std::u32;
+
+use microvmi::api::{EventReplyType, EventType, InterceptType, Introspectable};
+
+fn parse_args() -> ArgMatches<'static> {
+    App::new(file!())
+        .version("0.3")
+        .author("Mathieu Tarral")
```

Comment:
nitpick: Mathieu is not the author of this file, you are. Either remove this or enter your own name.